flight/Modules/Stabilization/stabilization.c

```diff
@@ -953,9 +954,23 @@ static void stabilizationTask(void* parameters)
 
 		for (int i = 0; i < MAX_AXES; i++) {
 			/* Solve for LQG, if it's configured for an axis. */
-			if (lqg[i] && !lqg_is_solved(lqg[i])) {
-				lqg_run_covariance(lqg[i], 1);
-				dump_lqg_solution(lqg[i], i);
+			if (lqg[i]) {
+				int status = lqg_solver_status(lqg[i]);
+				if (status == LQG_SOLVER_FAILED) {
+					/* Values don't converge in time, probably bogus. Light up the Christmas three. */
+					SystemAlarmsConfigErrorOptions err;
+					SystemAlarmsConfigErrorGet(&err);
+					if (err != SYSTEMALARMS_CONFIGERROR_LQG) {
+						err = SYSTEMALARMS_CONFIGERROR_LQG;
+						SystemAlarmsConfigErrorSet(&err);
+						AlarmsSet(SYSTEMALARMS_ALARM_SYSTEMCONFIGURATION, SYSTEMALARMS_ALARM_ERROR);
+					}
+				} else if (status == LQG_SOLVER_RUNNING) {
+					lqg_run_covariance(lqg[i], 1);
```

Comment:
Shouldn't the alarm be set here, too?

Reply:
OK, changed it. Works, too. With beta 6 and tau 250, the Seppuku needs a moment to converge, longer than for the GCS to connect, CONFIG is in error state and then clears up.